src/dataconnect/fileUtils.ts

```diff
@@ -106,3 +106,30 @@ export async function pickService(
   }
   return serviceInfo;
 }
+
+// case insensitive indicators for supported app platforms
+const WEB_INDICATORS = ["package.json", "package-lock.json"];
```

Comment:
Consider checking for a directory named "node_modules" as another potential signal